build.sbt

```diff
@@ -43,13 +41,8 @@ libraryDependencies ++= Seq(
     exclude("org.typelevel", "cats-laws_2.11")
     exclude("org.typelevel", "cats-kernel-laws_2.11"),
   "ch.qos.logback" % "logback-classic" % "1.2.1" % Provided,
-  "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
-  "io.spray" %% "spray-routing" % sprayV % Provided,
-  "io.spray" %% "spray-http" % sprayV % Provided,
-  "io.spray" %% "spray-can" % sprayV % Provided,
   "com.typesafe.akka" %% "akka-actor" % akkaV % Provided,
-  //---------- Test libraries -------------------//
```

Comment:
I thought I'd seen it before but went to cromwell for the source of truth when I removed it. I like moving ahead of the curve, as evidenced by removing spray code ;)